src/Bundle/Resources/config/services/routing.xml

```diff
@@ -22,6 +26,7 @@
             </argument>
             <argument>%kernel.environment%</argument>
             <tag name="routing.loader" />
+            <deprecated package="sylius/resource" version="1.13">The "%service_id%" service is deprecated since sylius/resource-bundle 1.13 and will be removed in sylius/resource-bundle 2.0. Use "sylius_resource.symfony.routing.loader.resource" instead.</deprecated>
```

Comment:
There is no such a service: `sylius_resource.symfony.routing.loader.resource` 🤔

Reply:
fixed, well observed.